src/Neo/Ledger/MemoryPool.cs

```diff
@@ -367,6 +367,12 @@ private bool CheckConflicts(Transaction tx, out List<PoolItem> conflictsList)
             // Step 2: check if unsorted transactions were in `tx`'s Conflicts attributes.
             foreach (var hash in tx.GetAttributes<Conflicts>().Select(p => p.Hash))
             {
+                // Firstly, check whether there's a free space in the list of transactions that has the same
+                // conflicts as `tx` has (in case of successfull verification, `tx` has to be added to this list).
+                if (_conflicts.TryGetValue(hash, out var pooled))
+                    if (pooled.Count() >= Neo.SmartContract.Native.LedgerContract.MaxAllowedConflictingTransactions) return false;
```

Comment:
@AnnaShaleva read the current stored ones?

Reply:
It's not necessary here, because we have a proper check during Conflicts attribute verification (the call to `CanFitConflictingTransaction`). So what we can get in the end is 16 transactions at max that already stored plus 16 transactions that may be accepted in the next block. After that attribute verification will fail for that conflict.